### ISSUE TITLE
Show full file name as tooltip in flame graph

### ISSUE THREAD

**Issue by mabuchner** [Viladoman/SeeProfiler#2]
If there is insufficient space in the flame graph, then file names will get truncated.

I know, that I can zoom in and zoom out, but I think, it would be useful, to get a tooltip with the complete file name when hovering with the cursor over a certain item.

**Comment by Viladoman** [comment]
I agree. Doing some mouse over in the timeline window has been in my 'to do' list for a while.
